packages/react-devtools-shared/src/backend/console.js

```diff
@@ -175,9 +176,19 @@ export function patch({
                     onErrorOrWarning(
                       current,
                       ((method: any): 'error' | 'warn'),
-                      args,
+                      // Copy args before we mutate them (e.g. adding the component stack)
+                      alreadyHasComponentStack
+                        ? [
+                            // Revert https://github.com/facebook/react/blob/cb141681750c8221ac799074df09df2bb448c7a4/packages%2Fshared%2FconsoleWithStackDev.js#L35
+                            // TODO (inline errors) This depends on the renderer version.
+                            args[0].replace(/%s$/, ''),
```

Comment:
This makes me a little uneasy. What if the component stack isn't the last substitution?

Reply:
See https://github.com/facebook/react/pull/20463#discussion_r544514148